requirements/reproducible-mac.txt

```diff
@@ -415,6 +415,10 @@ mypy-boto3-iam==1.28.37 \
     --hash=sha256:39bd5b8b9a48cb47d909d45c13c713c099c2f84719612a0a848d7a0497c6fcf4 \
     --hash=sha256:a5ed8c70c610f2ae7ee4a32ecf87c42824bfeb1e8a8c6347d888c73383d2c61f
     # via boto3-stubs
+mypy-boto3-kinesis==1.28.36 \
```

Comment:
Will these be updated by the other PR that you raised?

Reply:
Yes, I will merge that PR first and update this PR.